2.Views/WebAppPages/HomePage.cs

```diff
@@ -243,13 +243,13 @@ public class HomePage
     // DON'T USE THIS [DEPRECATED]
     // USE ProjectButton INSTEAD
     public IClickable ProjectTd(string projectName) =>
-    new Button(
-        "",
-        new Locator(
-            LocatorType.XPath,
-            $"//div[@id='ProjectListPlaceHolder']//td[text()='{projectName}']"
-        )
-    );
+        new Button(
+            "",
+            new Locator(
+                LocatorType.XPath,
+                $"//div[@id='ProjectListPlaceHolder']//td[text()='{projectName}']"
```

Comment:
Please follow the instructions in the comments above
```
// DON'T USE THIS [DEPRECATED]
// USE ProjectButton INSTEAD
```
Also, remove this method if unused.